src/Lime.Transport.WebSocket/WebSocketTransportListener.cs

```diff
@@ -1,265 +1,169 @@
 ﻿using System;
+using System.Buffers;
+using System.Linq;
 using System.Net;
 using System.Net.WebSockets;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
 using System.Threading;
+using System.Threading.Channels;
 using System.Threading.Tasks;
-using System.Threading.Tasks.Dataflow;
 using Lime.Protocol;
 using Lime.Protocol.Network;
 using Lime.Protocol.Serialization;
 using Lime.Protocol.Server;
-
-#if net461
-using SslCertBinding.Net;
-#endif
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
 
 namespace Lime.Transport.WebSocket
 {
-    /// <summary>
-    /// Defines a websocket transport listener service.
-    /// </summary>
-    /// <seealso cref="Lime.Protocol.Server.ITransportListener" />
     public class WebSocketTransportListener : ITransportListener
```

Comment:
I think it worth comment that this class uses Kestrel on its implementation

Reply:
done